### PR TITLE
Applied fixes from StyleCI

### PR DESCRIPTION
**Pull request by lsmith77**
This pull request applies code style fixes from an analysis carried out by [StyleCI](https://styleci.io).

---

For more information, click [here](https://styleci.io/analyses/87KJ08).

### CODE REVIEW

File: Controller/TwigExceptionController.php [hunk 1]
```diff
@@ -44,7 +44,7 @@ public function getTemplating()
     }
 
     /**
-     * {inheritDoc}
+     * {inheritDoc}.
```
Comment (xabbuh) [inline Controller/TwigExceptionController.php]: This looks strange. Isn't this an issue with StyleCI?

Comment (Ener-Getick) [inline Controller/TwigExceptionController.php]: I think that ``inheritdoc`` should not be uppercased.

Comment (sstok) [inline Controller/TwigExceptionController.php]: Correct version is `{@inheritdoc}` with the at-sign.

Comment (Ener-Getick) [inline Controller/TwigExceptionController.php]: :+1: @xabbuh fixed it in https://github.com/FriendsOfSymfony/FOSRestBundle/pull/1291

Comment (xabbuh) [inline Controller/TwigExceptionController.php]: Actually, I didn't even notice that the @ was missing. :)

Comment (Ener-Getick) [inline Controller/TwigExceptionController.php]: Me too, I noticed that when I saw your PR ^_^

**Pull request merged**